app/views/advocates/admin/advocates/_form.html.haml

```diff
@@ -1,27 +1,26 @@
 = form_for [:advocates, :admin, @advocate] do |f|
   - if @advocate.errors.any?
-    .validation-summary 
-      %h2.heading-small= "#{pluralize(@advocate.errors.count, 'error')} prohibited this advocate from being saved:"
-      %ul
-        - @advocate.errors.full_messages.each do |msg|
-          %li= msg
-    
+    %h2= "#{pluralize(@advocate.errors.count, 'error')} " + t("admin.form.errors")
+    %ul
+      - @advocate.errors.full_messages.each do |msg|
+        %li= msg
+
   %fieldset
     = f.fields_for :user do |user_fields|
-      = user_fields.label :email
+      = user_fields.label t("admin.form.user_email")
```

Comment:
This should be used as t('.email'), so the en.yml struture should be:

advocates:
    admin:
      advocates:
        form:
          email: Hi